### PR TITLE
Issue 14, Issue 81

### LINKED ISSUE #14: [FEA][NV] Add helper functions in Python API
Load adjacency list

and 

Add/delete :
- transposed adjacency list
- adjacency list

This is already implemented in the C++ API, just needs python bindings and simple tests.

### LINKED ISSUE #81: [BUG] delete cugraph/setup.[py/cfg]
They are outdated duplicates of the new ones in `cugraph/python` directory

### PR DESCRIPTION
**Pull request by jwyles**
Fixed issue in gdf_delete_adj_list, gdf_delete_edge_list, and gdf_delete_transpose that was causing double delete of user supplied columns. Re-enabled test of Graph.delete_xx methods. Deleted files for Issue 82. Close #14 Close #81

### CODE REVIEW

**Comment by GPUtester** [comment]
Can one of the admins verify this patch?

File: python/cugraph/graph/c_graph.pyx [hunk 1]
```diff
@@ -139,7 +139,7 @@ class Graph:
 
         return cudf.Series(src_data), cudf.Series(dest_data)
 
-    def to_edge_list(self):
+    def view_edge_list(self):
```
Comment (Iroy30) [inline python/cugraph/graph/c_graph.pyx]: isn't this a duplicate function name from line 119

Comment (jwyles) [reply python/cugraph/graph/c_graph.pyx]: Good catch, that function is duplicated. I will fix it. Thanks.

**Comment by raydouglass** [comment]
rerun tests

**Comment by raydouglass** [comment]
rerun tests

**Comment by afender** [comment]
Merging now in order to keep moving forward ... So that @jwyles can do his PR for #83 on top of it, so that I can start a PR for #92 (all these PR involve the same files).

**Pull request merged**